lib/net/ssh/authentication/ed25519.rb

```diff
@@ -135,6 +137,10 @@ def to_pem
             # TODO this is not pem
             ssh_type + Base64.encode64(@verify_key.to_bytes)
           end
+
+          def to_ssh
```

Comment:
could also override `to_s`.

This is the behavior of subclasses of OpenSSL::PKey::PKey, which are the other classes returned from `Net::SSH::KeyFactory::load_*_key`.